RELEASES.md

```diff
@@ -6,48 +6,68 @@ Below are the versioning schemes used for publishing debs:
 
 | Build target | Version Format |
 | -------- | -------- |
-| Devel series upstream release | XX.YY |
-| Devel series bugfix release | XX.YY.Z~ubuntu1|
-| Stable series release | XX.YY~ubuntu1~18.04.1|
-| [Daily Build Recipe](https://code.launchpad.net/~canonical-server/+recipe/ua-client-daily) | XX.YY+<revtime>-g<commitish>~ubuntu1~18.04.1 |
+| Devel series upstream release | `XX.YY` |
+| Devel series bugfix release | `XX.YY.Z~ubuntu1`|
+| Stable series release | `XX.YY~ubuntu1~18.04.1`|
+| [Daily Build Recipe](https://code.launchpad.net/~canonical-server/+recipe/ua-client-daily) | `XX.YY+<revtime>-g<commitish>~ubuntu1~18.04.1` |
 | Ubuntu PRO series release | binary copies of Daily PPA |
 
 ## Supported upgrade use-cases based on version formats
 
 | Upgrade path | Version diffs |
```

Comment:
This table isn't rendering in github, I think because its missing this line
```suggestion
| Upgrade path | Version diffs |
| -------- | -------- |
```